doc/rrdxport.pod

```diff
@@ -141,6 +145,33 @@ The resulting data section is:
     <row><t>1020615600</t><v>NaN</v><v>NaN</v></row>
   </data>
 
+All the statistics in the output will use the same step. The first sample will
+be the first sample starting at or immediately after B<--start>. The last
+sample will be the one ending at or immediately after B<--end>.
+
+Each sample has a timestamp and one or more values. The timestamps associated
+with a value in rrdtool ALWAYS represent the time the sample was taken. Since
+any value you sample will represent some sort of past state your sampling
+apparatus has gathered, the timestamp will always be at the end of the sampling
+period.
+
+Rrdtool does not store the actual samples, but does internal resampling of the
```

Comment:
Rrdtool -> RRDtool

Reply:
This was cut'n'pasted verbatim from [Tobi's comment](https://github.com/oetiker/rrdtool-1.x/issues/913#issuecomment-450800212), but I can see RRDtool is what is used also on the [home page](https://oss.oetiker.ch/rrdtool/). I'll fix that where you mention it and also in the paragraph above it.